Automattic/wp-calypso
# Domains Managment i1: Add domains sortable columns and sort domain name

Related to https://github.com/Automattic/dotcom-forge/issues/3423

## Proposed Changes

This PR adds the remaining domains columns both sortable and non-sortable. The intention was to add the sorting functions as part of this PR however the domain status sorting functions seem to be well involved as there's alot of moving parts, I think it's best done in another PR.

* Adds sortable columns
* Add column with property so we can specify the width of each column in percentage term.
* get fetched domain data so that sorting can be applied

## Testing Instructions



* go to `/domains/manage?flags=domains/management`
* Verify all the domain columns are present and sortable ones has the chevron
* Verify the domain name is sorting

![image](https://github.com/Automattic/wp-calypso/assets/47489215/8129bac2-d648-477e-98e3-3dcae3aa6957)


## Pre-merge Checklist
